deny.toml

```diff
@@ -18,7 +18,9 @@ allow = [
   "ISC",
   "MIT",
   "MIT-0",
+  "MPL-2.0",
```

Comment:
I don't think the MPL license can be allowed

https://www.mozilla.org/en-US/MPL/2.0/
> 3.2. Distribution of Executable Form
requires to distribute the source
> 3.3. Distribution of a Larger Work
even if it's just part of a larger work